pkg/vcs/vcs.go

```diff
@@ -8,13 +8,62 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"net/mail"
 	"regexp"
+	"sort"
 	"strings"
 	"time"
 
 	"github.com/google/syzkaller/pkg/osutil"
 )
 
+type RecipientType int
+
+const (
+	OpenList RecipientType = iota
+	Reviewer
+	FileList
```

Comment:
We need to express these types in terms that are interesting and meaningful to _users_ of this package, rather then in internal Linux kernel implementation detail terms. Otherwise we leak these uninteresting details to all users.
Consider, when dashboard will need to decide to put a particular recipient to To or Cc, it will need to know about all these 5 Linux kernel types and have a switch. And each user that will need to decide on To/Cc will need the same bit of knowledge. Now, if we support this for another operating system, which has own set of different recipient types, we will now need to update these switches to double-nested switches that know about all recipient types for all OSes. And every time something changes in any of the OS processes, we will need to update all of these places.

The only criteria that about outside of this package cares about is: if we want to add this recipient to To or Cc. This suggests that we should have 2 types and the decision for To/Cc should directly follow from the type. If we add another OS here, we will converge all of its types to these 2 abstract types interesting for users in this package. And then none of the users of this package need to be updated.